### ISSUE TITLE
[Docs] Improve Docs with the problem statement

### ISSUE THREAD

**Issue by niloysikdar** [leopardslab/react-email#115]
### Description:

- We need to describe the problem we're solving and also the solution on how our package is solving that particular problem.

### Outcome:

- Detailed writeup about the problem and the approach for the solution in the `README.md` file.

**Comment by Logan-kwan** [comment]
hi! if you could give me more information on the specific problems and solutions mentioned, I would love to work on this!

**Comment by niloysikdar** [comment]
Hi @Logan-kwan , thanks for showing interest to work on this issue. You can watch [this video](https://youtu.be/0vbLYyPerhU) to get complete info about the package.

Do let me know if you have any queries.

Also, make sure to drop a star on the repo to support the project and open-source🌟😇

**Comment by Logan-kwan** [comment]
just a quick question: do you have a preference on where to put the problem and suggested solution?

**Comment by niloysikdar** [comment]
Hmmm, I think it will be best to put it at the very start of the README after the name and description of the package.

**Comment by Anurag2293** [comment]
Hey @niloysikdar, would just like to ask if any help is further required for this issue. 
Would like to contribute to it?

**Comment by niloysikdar** [comment]
Sure @Anurag2293

**Comment by Anurag2293** [comment]
Can you guide me on how to get started?
I am a complete beginner in open source, though I am well versed in Git & GitHub.
Do I need to clone this project in my local device for the documentation part as well ?

**Comment by niloysikdar** [comment]
Yes @Anurag2293, first you need to create a fork of the repo and then clone it into your local machine.

Then go the the `/docusaurus` folder and enter `yarn` command to install the dependencies for the docs. After installing the dependencies, run `yarn start` to start the development server for docusaurus. Go to `localhost:3000` and you'll be able to see the docusaurus site up and running.

Do let me know if you have any queries or need any help. Don't forget to join the gitter channel for the discussions.

Also, make sure to drop a star on the repo to support the project and open-source🌟😇

**Comment by Anurag2293** [comment]
Hey @niloysikdar so I am done with the setup and docusaurus app running on my local device.
I would like to ask about any good first issues which I can work on. 

p.s. I am very new to open source. kindly bear with me 😅

**Comment by niloysikdar** [comment]
Awesome @Anurag2293, you can explore the Beginner Friendly issues [here](https://github.com/leopardslab/react-email/issues?q=is%3Aissue+is%3Aopen+label%3A%22Beginner+Friendly%22)

Do let me know if you need any more help.

**Comment by Anurag2293** [comment]
Hey @niloysikdar, after going through the first issues, I think I need to learn typescript for react (currently using plain JS only).
I will get back to you after learning React with TS in 1-2 days.

**Comment by niloysikdar** [comment]
Yes, you need to have the idea of TypeScrpt.
Go ahead, all the best.
